apps/website/src/pages/docs/tabs.mdx

```diff
@@ -9,27 +9,92 @@ group: core
 
 import PropsTable from '~/components/PropsTable.astro';
 import LiveExample from '~/components/LiveExample.astro';
-import Placeholder from '~/components/Placeholder.astro';
 import * as AllExamples from 'examples';
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='core-tabs' />
-
 <LiveExample src='Tabs.main.tsx'>
   <AllExamples.TabsMainExample client:load />
 </LiveExample>
 
-## Usage
+### Composition API
+
+You can build Tabs with subcomponents that are fully customizable.
+
+We provide:
+
+- `Tabs.Wrapper`
+- `Tabs.List`
+- `Tabs.Tab`
+- `Tabs.TabIcon`
+- `Tabs.TabLabel`
+- `Tabs.TabDescription`
+- `Tabs.Actions`
+- `Tabs.Panel`
+
+`Tabs.Tab` and `Tabs.Panel` can be associated with each other by passing them the same `value`.
+
+### Props API
+
+You can use Tabs directly with labels, actions, and children as panels. Although this pattern can suffice for simple cases, we strongly recommend using composition API for the best experience.
+
+A list of all available props can be found at the end of this [page](#props).
+
+<LiveExample src='Tabs.legacy.tsx'>
```

Comment:
if you go with the brief static code example for composition api, you can replace this live example with equivalent static code for legacy api. it will flow nicely with wording like this:

> The following code is equivalent to the composition example above:
> ```tsx
> <Tabs
>   labels={[
>     <Tab key='apple' label='Apple' />,
>     <Tab key='orange' label='Orange' />,
>     <Tab key='pear' label='Pear' />,
>   ]}
> >
>   {/* … content for active tab */}
> </Tabs>
> ```

Reply:
Looks good!